ztp/gitops-subscriptions/argocd/README.md

```diff
@@ -3,6 +3,14 @@
 
 ## Installing the GitOps Zero Touch Provisioning pipeline
 
+### Obtaining the ZTP site generator container
+The GitOps ZTP infrastructure relies on the ztp-site-generator container to provide the tools which transform SiteConfig and PolicyGenTemplate CRs into the underlying installation and configuration CRs. This container can be pulled from pre-build/official sources or built from source by following [Building the container](../../resource-generator/README.md)
+
+## Obtaining pre-built image
+```
+    $ podman pull quay.io/redhat_emp1/ztp-site-generator:latest
```

Comment:
I like this organization a lot.  Because of it, we can remove the `pull` part of step "2" in the "Preparation of ZTP GIT repository" below.

Reply:
IE, remove https://github.com/openshift-kni/cnf-features-deploy/pull/940/files#diff-a5b2fbb23a415e4c707c5d404ed7656ef83e319e6725d52f86be1c9d0e927021R19

Reply:
Good point.

Reply:
done